src/orga/governance.rst

```diff
@@ -85,9 +85,13 @@ Standard items are ones where public record and discourse is
 preferable. Sensitive voting items are ones where the results of the
 vote should remain private to the voters after the vote has occurred.
 Sensitive votes should take place on a secure anonymous voting platform
-(such as `Polys <polys.me>`_ or
-`the Helios voting system <https://vote.heliosvoting.org/>`_) in order to
-retain anonymity.
+in order to retain election integrity and anonymity. (We have used `Polys <polys.me>`_ 
+and `the Helios voting system <https://vote.heliosvoting.org/>`_, but are 
+open to any secure, anonymous system.) The email capability of your chosen 
+voting platform should be used for sending voting invitations and reminders,
+and you should use the email list from
+https://github.com/conda-forge/conda-forge.github.io/blob/master/src/core.csv as
+the authoritative list of emails to use.
 
 The default voting period is 1 week (7 days). This may be modified at
 the time when a vote is called, but may never be less than 24 hrs.
```

Comment:
```suggestion
the time when a vote is called, but may never be less than 24 hrs.

Additional requirements may apply in case low turnouts have to be handled, see "Quorum" below.
```
^-- feel free to change the phrasing as you see fit.

This is just to remind people to plan ahead if they want timeouts to happen -> https://github.com/conda-forge/conda-forge.github.io/pull/970/commits/37f109730bb521186190c328045af1ea8a03de4b#diff-aeeef496313214bb0e1ff8c044612ad3R139 .